README.md

```diff
@@ -85,6 +85,18 @@ To learn more about the platform, check out our [documentation](https://github.c
   ./run_sc_component_manager.sh -c ../../ostis-web-platform.ini -b ../../repo.path
   ```
 
+- Natively (using sc-component-manager)
+
+  ```sh
+  git clone https://github.com/ostis-ai/ostis-web-platform
+  cd ostis-web-platform/scripts
+  git checkout feature/component_manager
+  ./install_component_manager_platform.sh
+  ./build_kb.sh
+  cd ../sc-machine/scripts
+  ./run_sc_component_manager.sh -c ../../ostis-web-platform.ini
```

Comment:
I got error after the last step
![image](https://github.com/ostis-ai/ostis-web-platform/assets/8680298/23bc7c9a-b120-42c4-93d0-770925642e16)

Reply:
seems like you don't have svn. Run `sudo apt install subversion` please